qdrant-landing/content/documentation/cloud/configure-cluster.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Configure Clusters
+weight: 55
+---
+
+# Configure Qdrant Cloud Clusters
+
+Qdrant Cloud offers several advanced configuration options to optimizer your clusters to your specific needs. You can access these options from the Cluster details page in the Qdrant Cloud console.
```

Comment:
Cluster details be full capitalised (Cluster Details) as it's the name of the page